scripts/ceph-website/install-deps.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -ex
+
+# install node.js
+NODEJS_MAJOR_VERSION=20
+DISTRO="$(lsb_release -cs)"
+if [[ ! $(command -v node) || $(node --version | grep -oE "v([0-9])+" | cut -c 2-) < ${NODEJS_MAJOR_VERSION} ]]; then
+    NODEJS_KEYRING=/usr/share/keyrings/nodesource.gpg
```

Comment:
I would add an 'echo' here explaining what the script found and what it's doing ("nodejs version %d too old, installing %d instead")